### PR TITLE
Add conversion between different anomalies in hyperbolic orbits

### PR DESCRIPTION
**Pull request by anhiga**
Current anomaly conversions (mean -> true, true -> eccentric, ...)
are only valid when eccentricity < 1, not for hyperbolic orbits.

### CODE REVIEW

**Comment by anhiga** [comment]
You are right, all the functions in the `angles` module accept float instead of quantities, but in the corresponding tests, they all are used with quantities, without problems.

However, some of the new functions can't accept both types. Should they accept float then? Or should we change all the functions to work with quantities?

File: src/poliastro/twobody/angles.py [hunk 1]
```diff
@@ -60,6 +93,27 @@ def E_to_nu(E, ecc):
...
+    ecc : float
+        Eccentricity (>1).
+
+    Returns
+    -------
+    nu : float
+        True anomaly (rad).
+
+    """
+    nu = 2 * np.arctan((np.exp(F.value) * np.sqrt(ecc + 1) - np.sqrt(ecc + 1)) /
```
Comment (Juanlu001) [inline src/poliastro/twobody/angles.py]: Here is the problem with the units of `F`, right?

**Comment by Juanlu001** [comment]
All the tests pass with this diff:

```diff
diff --git a/src/poliastro/twobody/angles.py b/src/poliastro/twobody/angles.py
index ccb6ed3..3f50412 100644
--- a/src/poliastro/twobody/angles.py
+++ b/src/poliastro/twobody/angles.py
@@ -109,8 +109,8 @@ def F_to_nu(F, ecc):
         True anomaly (rad).
 
     """
-    nu = 2 * np.arctan((np.exp(F.value) * np.sqrt(ecc + 1) - np.sqrt(ecc + 1)) /
-                       (np.exp(F.value) * np.sqrt(ecc - 1) + np.sqrt(ecc - 1)))
+    nu = 2 * np.arctan((np.exp(F) * np.sqrt(ecc + 1) - np.sqrt(ecc + 1)) /
+                       (np.exp(F) * np.sqrt(ecc - 1) + np.sqrt(ecc - 1)))
     return nu
 
 
```

So I'm not sure what the problems are.

**Comment by Juanlu001** [comment]
Oh, I tried to implement `test_mean_to_true_hyperbolic` and now I see. Working on it.

**Comment by Juanlu001** [comment]
Done! Please review my changes to your changes :sweat_smile:

**Comment by anhiga** [comment]
The problem was that `np.exp()` raised an error when used with quantities, but I can't try it now, and tests are passing now, so...

**Comment by Juanlu001** [comment]
Exactly, the error was fixed using the "angles are dimensionless" trick as it's used in other functions.

**Comment by anhiga** [comment]
Great then, merging and closing #275

**Pull request merged**